boilerplate/frontend/supertokens-react-custom/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
Most of it looks good. But i would need the following changes before merging:
- Better styling
- Removing of "react-custom" id. We should keep frontend as react ID and use the ui type flag in our logic.